site/gdocs/components/KeyInsights.test.tsx

```diff
@@ -7,17 +11,16 @@ import {
     slugify,
 } from "@ourworldindata/utils"
 import { fireEvent, render, screen } from "@testing-library/react"
-import "@testing-library/jest-dom"
+import "@testing-library/jest-dom/vitest"
```

Comment:
Despite jest being in the name, [this really is the way](https://testing-library.com/docs/svelte-testing-library/setup/#vitest) to setup testing-library for vitest.